### ISSUE TITLE
Smooth lighting bug with 1.16.3 version

### ISSUE THREAD

**Issue by FarmerMisShelley** [sp614x/optifine#4867]
Sorry if I use the wrong format but I couldn't find a format post. 

I was messing around with my settings to optimize a few things. I turned off smooth lighting and it was almost like I suddenly had an indefinite form of the night vision effect. I told a friend to test it since I was also using a texture pack so it coulda been because of that and they said they had the same issue. While this isn't something I am necessarily upset by myself, its still a bug and it still should be reported. 

![2020-09-29_14 00 46](https://user-images.githubusercontent.com/43741916/94615299-646c5480-025c-11eb-8103-5f0c20187ce8.png)
![2020-09-29_14 01 29](https://user-images.githubusercontent.com/43741916/94615304-659d8180-025c-11eb-937f-e1d159310acb.png)
![2020-09-29_14 01 41](https://user-images.githubusercontent.com/43741916/94615322-6b936280-025c-11eb-970a-216f455b0f3e.png)

**Comment by BlueStone231** [comment]
Yeah, I'm the "friend" mentioned here. We did some testing and found that the smooth lighting night vision was a consistent bug
i tested it with my texture packs both on and off and yeah, it's definitely an optifine issue.

**Comment by Rakete1111** [comment]
I had the same issue, but I had Smooth Lighting to Maximum. Changing it and playing with the slider didn't change anything. But what fixed the permanent night vision for me was changing the brightness to moody and then immediately back to bright.
